### PR TITLE
MongoDB atlas setup

### PR DESCRIPTION
**Pull request by bhaskarbhakat**
## Description
This PR is to connect mongo db.

## Implementation
- added config file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,12 +15,16 @@ app.use(
...
 app.use(express.json());
 
 app.use("/api", userValidationRoutes);
 app.use("/api", contestRankingRoutes);
 
-app.listen(port, () => {
-  console.log(`BACKEND IS RUNNING`);
-});
+app.listen(process.env.PORT, () => {
+  console.log(`Server is working on http://localhost:${process.env.PORT}`);
```
Comment (anaysarkar7) [inline index.js]: port number should not be console logged for production server

File: config/config.env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+PORT=8000
+MONGODB_URI=mongodb+srv://coderscalendar:WNCqy9WaUsWm8TmA@coders-calendar.juqlv2x.mongodb.net/?retryWrites=true&w=majority
```
Comment (anaysarkar7) [inline config/config.env]: add this file in gitignore URI exposed

File: config/database.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import mongoose from "mongoose";
+
+const connectDatabase = () => {
+  mongoose.set('strictQuery', false);
+  mongoose.connect(process.env.MONGODB_URI)
+    .then((data) => {
+      console.log(`Mongodb connected with server: ${data.connection.host}`);
```
Comment (anaysarkar7) [inline config/database.js]: Just log simply DB connected ?

**Review by anaysarkar7 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (bhaskarbhakat) [reply index.js]: Replaced

**Review by anaysarkar7 — APPROVED** [review]

**Pull request merged**